core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/ReadParser.scala

```diff
@@ -57,8 +57,10 @@ object ReadParser extends Logging {
     val locationTag = "Location: "
     val location = if (node.desc.contains(locationTag)) {
       val stringWithBrackets = getFieldWithoutTag(node.desc, locationTag)
-      // Remove prepended InMemoryFileIndex[ and return only location
-      // Ex: InMemoryFileIndex[hdfs://bdbl-rpm-1106-57451/numbers.parquet
+      // Remove prepended InMemoryFileIndex[  or  PreparedDeltaFileIndex[
+      // and return only location
+      // Ex: InMemoryFileIndex[hdfs://bdbl-rpm-1106-57451/numbers.parquet,
+      // PreparedDeltaFileIndex[file:/tmp/deltatable/delta-table1]
       stringWithBrackets.split("\\[", 2).last
```

Comment:
please also check that this exists before assuming it does... if it doesn't I'd say just put the entire thing in there

Reply:
done